# Latched message from SpeakEasy service to communicate
# its available:
#    - Text-to-speech engine names (e.g. ['festival', 'cepstral'])
#    - Voices for each text-to-speech engine (e.g. ['David'])
#    - Sound files (e.g. ['drill.wav', 'bomb.ogg'])
#
string[] ttsEngines
TtsVoices[] voices
string[] sounds

